document/CoreFormat.mdk

```diff
@@ -355,12 +359,52 @@ An `issue` object describes a single issue detected by a static analysis tool.
 ### `ruleId` property { #issue-ruleId }
 
 An `issue` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was violated.
+the stable, opaque identifier for the rule that was violated.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #issue-kind }
+
+An `issue` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify what kind of issue this is.
+
+If present, the `kind` property shall have one of the following values, with the specified meanings:
+
+* `warning`: The rule specified by the `ruleId` property (&sect;[#issue-ruleId]) was evaluated and a violation was found.
+
+* `error`: The rule specified by the `ruleId` property was evaluated and a serious violation was found.
+
+* `pending`: The rule specified by the `ruleId` property was evaluated, but whether the result of the evaluation
+constitutes a violation cannot be determined without further evaluation, either by a person or by an automated tool.
+
+~ IsoNote
+In some cases, team policy or corporate policy might determine if an analysis result constitutes a violation.
```

Comment:
As we discussed: Producing information about `pending` issues is valuable even if there is no automated way of applying policy to them and converting them into `warning` or `error` issues. A person can query for `pending` issues and apply policy "manually". In future, we could imagine enhancing the format to provide additional information that an automated tool could use to make the policy decision.

I will add the example you provided here.